### PR TITLE
Fix edit and onboard user

### PR DESCRIPTION
**Pull request by MohammadPCh**
- Giveth/giveth-io-typescript#68

### CODE REVIEW

File: pages/account.tsx [hunk 1]
```diff
@@ -36,29 +29,4 @@ const UserRoute: FC<IUserRouteProps> = ({ user }) => {
...
-		user = userData?.userByAddress;
-	}
-
-	return {
-		props: {
-			user: user,
-		},
-	};
-};
-
```
Comment (mateodaza) [inline pages/account.tsx:63]: We need this for my account page, why did you delete?

**Review by mateodaza — CHANGES_REQUESTED** [review]

**Review by mateodaza — CHANGES_REQUESTED** [review]

File: pages/account.tsx [hunk 2]
```diff
@@ -36,29 +29,4 @@ const UserRoute: FC<IUserRouteProps> = ({ user }) => {
...
 export default UserRoute;
```
[the remark this answers is not in the document]
Comment (mateodaza) [reply pages/account.tsx:32]: I missed you moved it! OK!
Then it's fine :)

[the remark this answers is not in the document]
Comment (MohammadPCh) [reply pages/account.tsx:32]: Thank you @mateodaza

**Comment by MohammadPCh** [comment]
@mateodaza I resolved the conflict with the develop branch, and it's ready to merge.
but I was forced to change some parts of the code because of conflict. could you please check the profile and account route again?
Thank you.
